Cmdr/BuiltInCommands/Utility/runif.lua

```diff
@@ -4,6 +4,66 @@ local conditions = {
 			return text:sub(#arg + 1)
 		end
 	end,
+
+	contains = function(text, arg)
+		if text:find(arg, 1, true) then
+			return text
+		end
+	end,
+
+	endsWith = function(text, arg)
+		if text:sub(-#arg) == arg then
+			return text:sub(1, -#arg - 1)
+		end
+	end,
+
+	pattern = function(text, arg)
+		if text:match(arg) then
+			return text
+		end
+	end,
+
+	equals = function(text, arg)
+		if text == arg then
+			return text
+		end
+	end,
+
+	notEquals = function(text, arg)
+		if text ~= arg then
+			return text
+		end
+	end,
+
+	greaterThan = function(text, arg)
+		if tonumber(text) > tonumber(arg) then
```

Comment:
At the moment you'd have to do `runIf greaterThan 1 2 command` and it would return true as 2 is greater than 1 but I think `tonumber(text) > tonumber(arg)` should be switched to `tonumber(arg) > tonumber(text)` to be a little more intuitive as you're comparing the argument against the text